### PR TITLE
Remove duplicate words - doc bug

### PR DESCRIPTION
**Pull request by TakaSoap**

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/whats-new/2020/01/devtools.md [hunk 1]
```diff
@@ -367,4 +367,4 @@ This work is licensed under a [Creative Commons Attribution 4.0 International Li
 [CCA4IL]: https://creativecommons.org/licenses/by/4.0  
 [CCby4Image]: https://i.creativecommons.org/l/by/4.0/88x31.png  
 [GoogleSitePolicies]: https://developers.google.com/terms/site-policies  
-[KayceBasques]: https://developers.google.com/web/resources/contributors/kaycebasques  
\ No newline at end of file
+[KayceBasques]: https://developers.google.com/web/resources/contributors/kaycebasques
```
Comment (TakaSoap) [inline microsoft-edge/devtools-guide-chromium/whats-new/2020/01/devtools.md:370]: Nothing changed here

File: microsoft-edge/devtools-guide-chromium/whats-new/2020/01/devtools.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ Many developers use other developer tools, like StackOverflow and VS Code, in th
 
 | | |
 | -------- | ---- |
-| Chinese (Simplified) - 中文(简体)（简体）| Chinese (Traditional) - 中文(繁體)（繁體）|
+| Chinese (Simplified) - 中文（简体）| Chinese (Traditional) - 中文（繁體）|
```
Comment (TakaSoap) [inline microsoft-edge/devtools-guide-chromium/whats-new/2020/01/devtools.md:57]: Remove duplicate words.

**Comment by jm-trd-ms** [comment]
Updated content in PR #619.

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**